.github/workflows/ci.yml

```diff
@@ -6,6 +6,7 @@ jobs:
   test:
     name: Test (Ruby ${{ matrix.ruby }})
     strategy:
+      fail-fast: false
       matrix:
         ruby:
```

Comment:
Thanks for the changes @PikachuEXE 🍻 

I see that rspec-rails has removed support for ruby below 2.5 https://github.com/rspec/rspec-rails/blob/main/Changelog.md#600--2022-10-10, should we remove those versions as well from here ?

Reply:
As long as you are fine I will remove it~

Reply:
Should we add 3.1 too?

Reply:
sure, we can do that, thanks!